components/sst_importer/src/sst_importer.rs

```diff
@@ -27,12 +31,17 @@ use crate::metrics::*;
 /// SSTImporter manages SST files that are waiting for ingesting.
 pub struct SSTImporter {
     dir: ImportDir,
+    key_manager: Option<Arc<DataKeyManager>>,
 }
 
 impl SSTImporter {
-    pub fn new<P: AsRef<Path>>(root: P) -> Result<SSTImporter> {
+    pub fn new<P: AsRef<Path>>(
+        root: P,
+        key_manager: Option<Arc<DataKeyManager>>,
```

Comment:
We can avoid sst_importer depends on encryption by `add_file` or `link_file` via `EncryptedEnv`, if the change is too large, we can postpone it.

Reply:
I think we need to abstract `Env` out and use it consistently in tikv for file operations. Let's postpone for later.